.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ if you have questions about contributing.
 
 <!-- Please give a short brief about these changes. -->
 
-- [ ] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/CONTRIBUTING.html)
+- [ ] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contrbuting.html)
```

Comment:
This link doesn't work either, there is a typo in the word `contributing`.

Reply:
Oops. Fixed.